AppCenter/AppCenter/Internals/Ingestion/MSHttpIngestion.h

```diff
@@ -25,6 +25,11 @@ static NSString *const kMSETagRequestHeader = @"If-None-Match";
  */
 @property(nonatomic) NSURL *sendURL;
 
+/**
+ * Http verb
+ */
+@property(nonatomic) NSString *httpVerb;
```

Comment:
The best practice is to use the `copy` attribute for properties of type `NSString`.